base/libgit2/types.jl

```diff
@@ -274,20 +274,20 @@ immutable DiffFile
     id::Oid
     path::Cstring
     size::Int64
-    flags::Cuint
```

Comment:
`Cuint` is `UInt32`,  see http://docs.julialang.org/en/latest/manual/calling-c-and-fortran-code/#mapping-c-types-to-julia